frontend/src/components/header/HeaderBar.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <header>
-    <img v-if="showLogo !== undefined" :src="logoURL" />
+    <a href="/">
+      <img v-if="showLogo !== undefined" :src="logoURL" alt="logo" />
+    </a>
```

Comment:
I think it should be
```suggestion
    <a v-if="showLogo !== undefined" href="/">
      <img :src="logoURL" alt="logo" />
    </a>
```